Adding an explanation how to use `htpasswd` for k8s secret (#6194)
containous/traefik: docs/content/middlewares/basicauth.md

    secret: authsecret

---
# Note: in a kubernetes secret the string (e.g. generated by htpasswd) must be base64-encoded first.

> # To create user:password pair, it's possible to use this command:

Reviewer: ```suggestion
# To create an encoded user:password pair, the following command can be used:
```
